File: 250408/더 긴 문자열/longer-string.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        String str1 = sc.next();
        String str2 = sc.next();
        if(str1.length() > str2.length()) {
            System.out.printf("%s %d", str1, str1.length());
        }
        else if(str1.length() < str2.length()) {
            System.out.printf("%s %d", str2, str2.length());
        }
        else {
            System.out.print("same");
        }
    }
}

File: 250408/더 긴 문자열/longer-string.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    string str1, str2;
    cin >> str1 >> str2;
    if(str1.length() > str2.length()) {
        cout << str1 << " " << str1.length();
    }
    else if(str1.length() < str2.length()) {
        cout << str2 << " " << str2.length();
    }
    else {
        cout << "same";
    }
    return 0;
}